### PR TITLE
Removed atomicFu plugin

### PR DESCRIPTION
**Pull request by aclassen**
Added iosSimulator
Removed kmmBridge and pod creation

Please see our contributing guidelines (contributing.md) primarily make sure to sign our cla as we cannot accept code externally without a signed cla

https://opensource.dropbox.com/cla/

### CODE REVIEW

**Comment by matt-ramotar** [comment]
Thanks for opening @aclassen! What is the reason for removing KMM Bridge? Our CI uses KMM Bridge to generate a Swift package on release

File: cache/build.gradle.kts [hunk 1]
```diff
@@ -90,14 +82,6 @@ mavenPublishing {
     signAllPublications()
 }
 
-addGithubPackagesRepository()
-kmmbridge {
```
Comment (matt-ramotar) [inline cache/build.gradle.kts:94]: Thanks for opening @aclassen! What is the reason for removing KMM Bridge? Our CI uses KMM Bridge to generate a Swift package on release

Comment (aclassen) [reply cache/build.gradle.kts:94]: So the idea is to use store direct as pod in objc ?

Comment (matt-ramotar) [reply cache/build.gradle.kts:94]: We want to make `cache` and `store` available for iOS folks

Comment (aclassen) [reply cache/build.gradle.kts:94]: So store should be available as a regular pod  , without any shared code ?

Comment (matt-ramotar) [reply cache/build.gradle.kts:94]: Not sure I am understanding - What do you mean by "shared code"?

Comment (matt-ramotar) [reply cache/build.gradle.kts:94]: I don't have an opinion on `multicast` but think it makes sense to make `cache` available as a Swift package. Without any question we need to make `store` available as one.

Comment (aclassen) [reply cache/build.gradle.kts:94]: What i mean is, do you want to be able to the following : open xcode -> create a new ios project -> add store as pod in your podfile and use it in swift.

Comment (matt-ramotar) [reply cache/build.gradle.kts:94]: Yes

Comment (aclassen) [reply cache/build.gradle.kts:94]: oh ok

Comment (matt-ramotar) [reply cache/build.gradle.kts:94]: Thanks 🙏🏽

Comment (aclassen) [reply cache/build.gradle.kts]: @matt-ramotar Can you maybe help me to fix the failing workflow ?

Comment (matt-ramotar) [reply cache/build.gradle.kts]: Mind adding `iosSimulatorArm64` in a separate PR? We will need to modify our CI and I unfortunately won't be able to work on this until next week sometime. See #511

**Comment by aclassen** [comment]
i close this as my commits are signed with wrong email

**Pull request closed without merging**